core/src/main/java/psiprobe/AbstractTomcatContainer.java

```diff
@@ -406,7 +406,7 @@ public void listContextJsps(Context context, Summary summary, boolean compile) {
   }
 
   @Override
-  public org.apache.juli.logging.Log getLogger(Context context) {
+  public Object getLogger(Context context) {
```

Comment:
What motivated this change? All supported Tomcat versions return `org.apache.juli.logging.Log`.